server/pairing/sync_device_test.go

```diff
@@ -733,40 +729,23 @@ func defaultSettings(generatedAccountInfo generator.GeneratedAccountInfo, derive
 func nodeConfigForLocalPairSync(installationID, keyUID, tmpDir string) (*params.NodeConfig, error) {
 	// Set mainnet
 	nodeConfig := &params.NodeConfig{}
+	nodeConfig.LogEnabled = true
 	nodeConfig.LogLevel = "DEBUG"
+	nodeConfig.LogDir = tmpDir
 	nodeConfig.KeyStoreDir = filepath.Join(keystoreDir, keyUID)
 	nodeConfig.KeycardPairingDataFile = filepath.Join("keycard", "pairings.json")
-	nodeConfig.Name = "StatusIM"
+	nodeConfig.ShhextConfig = params.ShhextConfig{
+		InstallationID: installationID,
+	}
+
+	// need specify cluster config here, otherwise TestPairingThreeDevices will fail due to no messages(CR) received
+	// TODO(frank) need to figure out why above happen
 	clusterConfig, err := params.LoadClusterConfigFromFleet("eth.prod")
```

Comment:
Let's replace `eth.prod` with constant?
https://github.com/status-im/status-go/blob/34c693bdeae4de261e42d8c935c0fb8a4681f4a0/params/cluster.go#L6

Or even better take one of supported fleets:
https://github.com/status-im/status-go/blob/34c693bdeae4de261e42d8c935c0fb8a4681f4a0/params/cluster.go#L24-L29